cli/template/base/_eslintrc.cjs

```diff
@@ -6,7 +6,7 @@ const config = {
   },
   plugins: ["@typescript-eslint"],
   extends: [
-    "plugin:@next/next/recommended",
```

Comment:
Isn't there `@next/next/core-web-vitals`? I don't think renovate for example bumps eslint-config-next at the same time as next (but using @next/*` will. Just a nit thing